package.json

```diff
@@ -9,7 +9,7 @@
   "devDependencies": {
     "angular-mocks": "~1.5.11",
     "bower": "^1.8.0",
-    "gulp": "^3.9.1",
+    "gulp": "~4.0.0",
```

Comment:
Can you also check in a modified `yarn.lock`. It will be generated, if you run yarn instead of npm.